packages/graph/hash_graph/tests/integration/postgres/mod.rs

```diff
@@ -252,16 +298,34 @@ impl DatabaseApi<'_> {
 
     pub async fn get_link_target(
         &self,
-        source_entity_id: EntityId,
+        source_entity: EntityId,
         link_type_uri: VersionedUri,
     ) -> Result<Outgoing, QueryError> {
-        self.store
-            .get_link_target(source_entity_id, &link_type_uri)
-            .await
+        Ok(self
+            .store
+            .get_links(
+                &LinkQuery::new()
+                    .with_source_entity_id(source_entity)
+                    .with_link_types(|link_types| {
+                        link_types
+                            .with_uri(link_type_uri.base_uri())
+                            .with_version(link_type_uri.version())
+                    }),
+            )
+            .await?
+            .pop()
+            .expect("No links found")
```

Comment:
Would be only a single link
```suggestion
            .expect("No link found")
```